cfme/tests/control/test_basic.py

```diff
@@ -46,6 +46,47 @@
 ]
 
 
+ALERT_PROFILE_FIXTURE_PARAMS = [
+    {"alert_profile_class": explorer.ClusterAlertProfile,
+     "alert_profile_type": "Cluster / Deployment Role"},
+    {"alert_profile_class": explorer.DatastoreAlertProfile,
+     "alert_profile_type": "Datastore"},
+    {"alert_profile_class": explorer.HostAlertProfile,
+     "alert_profile_type": "Host / Node"},
+    {"alert_profile_class": explorer.ProviderAlertProfile,
+     "alert_profile_type": "Provider"},
+    {"alert_profile_class": explorer.ServerAlertProfile,
+     "alert_profile_type": "Server"},
+    {"alert_profile_class": explorer.VMInstanceAlertProfile,
+     "alert_profile_type": "VM and Instance"}
+]
```

Comment:
We want the classes to know their textual type representation so that we dont have to add mappings like this one into test cases. Discussed offline.